src/makeInfinityAdder.js

```diff
@@ -5,8 +5,24 @@
  * @return {function}
  */
 
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(initialValue = 0) {
```

Comment:
Consider removing the default value for the 'initialValue' parameter, as it is not mentioned in the task description.